### DISCUSSION TITLE
startup.sh add command

### DISCUSSION THREAD

**Discussion by hrpoint** [Arkady23/V380-Web#26] [Ideas]
Is it possible to add command to startup?

I want to disable wifi:
 /sbin/rmmod atbm603x_wifi_HT20_usb

and have edited /mnt/sdcard/ark-add-on/startup.sh
but it doesnt work

**Comment by Arkady23** [comment]
Of course you can. This is a command bash file. To see why the command is not running, you need to include debug output in some message file.
